miso-web/src/main/webapp/scripts/list_container.js

```diff
@@ -50,7 +50,7 @@ ListTarget.container = {
         },
         ListUtils.idHyperlinkColumn("Last Run Name", "run", "lastRunId",
             function(container) {
-              return "RUN" + container.lastRunId;
+              return (container.lastRunId ? "RUN" + container.lastRunId : null);
```

Comment:
That's what my change in `lims.js#289` is supposed to do, and is showing as no text on my machine. Are you seeing text when you deploy it?